src/virtualship/utils.py

```diff
@@ -97,11 +97,11 @@ def mfp_to_yaml(excel_file_path: str, yaml_output_path: str):  # noqa: D417
     coordinates_data = coordinates_data.dropna()
 
     # maximum depth (in meters), buffer (in degrees) for each instrument
-    instrument_properties = {
-        "XBT": {"maximum_depth": 2000, "buffer": 1},
-        "CTD": {"maximum_depth": 5000, "buffer": 1},
-        "DRIFTER": {"maximum_depth": 1, "buffer": 5},
-        "ARGO_FLOAT": {"maximum_depth": 2000, "buffer": 5},
+    instrument_max_depths = {
+        "XBT": 2000,
+        "CTD": 5000,
+        "DRIFTER": 1,
+        "ARGO_FLOAT": 2000,
     }
```

Comment:
@VeckoTheGecko 
Wasn't better to keep the buffer? The users will have to add this manually?

Reply:
Heyo! Sorry if I forgot to explain this in another message (its been a bit of a hackathon here to get things good to go). We were thinking of having the buffering in the actual data download, and not the specification of the spacetime region (so the spacetime region then becomes the region of interest). This would mean that we can reduce the data footprint by looking on a per-dataset level

cc @ammedd